com/win32com/server/register.py

```diff
@@ -172,6 +173,8 @@ def RegisterServer(clsid,
      clsctx -- One of the CLSCTX_* constants.
      addnPath -- An additional path the COM framework will add to sys.path
                  before attempting to create the object.
+     exeOption -- Option under which the server shall be started. By detault
+                  this varaible is set to '/Automate'.
```

Comment:
typo in "varaible"

Reply:
Fixed in https://github.com/mhammond/pywin32/pull/1130/commits/8f287ff5919a98edc1cacd58111aa30c661256a5